pkgs/development/python-modules/mcuuid/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "mcuuid";
+  version = "1.1.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "clerie";
+    repo = "mcuuid";
+    rev = "refs/tags/${version}";
+    hash = "sha256-YwM7CdZVXpUXKXUzFL3AtoDhekLDIvZ/q8taLsHihNk=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  doCheck = false;
```

Comment:
Can you add a reason for disabling the test?

Reply:
Oh sure, I do